notification-portlet-webcomponents/notification-modal/package.json

```diff
@@ -59,10 +65,5 @@
       "vue-cli-service lint",
       "git add"
     ]
-  },
-  "babel": {
```

Comment:
I'm surprised this works without the babel config.
![](https://media.giphy.com/media/R6bDgXEXCLcIw/giphy.gif)